kolibri/core/templates/kolibri/base.html

```diff
@@ -7,6 +7,7 @@
   {% base_frontend_async %}
   {% base_frontend_sync %}
   <link type="text/css" rel="stylesheet" href="{% static 'core/css/components.css' %}" />
+  <link type="text/css" rel="stylesheet" href="{% static 'core/css/kolibri_style.css' %}" />
```

Comment:
Once you've installed the style loader, you can just `require` the CSS files from within your primary JS plugin file. So for example, I'm using normalize.css. I run:

```
npm install style-loader css-loader normalize-css --save-dev
```

and then in my main js file I have:

```
require('normalize-css');
```

If this were a local file like yours, I think you could reference it like

```
require('./kolibri_style.css');
```

And of course, if it's LESS or SASS or whatever you can install the right loader for webpack.